hw/formal/data/fpv.mk

```diff
@@ -0,0 +1,44 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+.DEFAULT_GOAL := all
+
+all: build
+
+###################
+## build targets ##
+###################
+build: compile_result
+
+gen_sv_flist:
+	@echo "[make]: gen_sv_flist"
+	cd ${build_dir} && ${sv_flist_gen_cmd} ${sv_flist_gen_opts}
+
+pre_compile: gen_sv_flist
+	@echo "[make]: pre_compile"
+	mkdir -p ${build_dir} && env | sort > ${build_dir}/env_vars
+	-cp -Ru ${tool_srcs} ${sv_flist_gen_dir}
```

Comment:
Why is the return code from `cp` being ignored here? I would have thought we wanted to see the error if we don't manage to copy a file we need?